app/views/shops/show.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1><%= @shop.name %></h1>
+<p><%= @shop.address %></p>
+<p><%= @shop.city %>, <%= @shop.state %> <%= @shop.zip %></p>
+
+<%= link_to 'Back to All Shops', shops_path %>
```

Comment:
I appreciate moving quickly on this, but we'll just have to refactor this to react on the next feature anyways